sdk/nodejs/provisioning/test/provisioner.spec.ts

```diff
@@ -1,19 +1,19 @@
-import assert from 'assert';
-import { getProvisioner } from '../provisioner.js';
-import { DEFAULT_HOST } from '../default.js';
+import assert from "assert";
+import { getProvisioner } from "../provisioner.js";
+import { DEFAULT_HOST } from "../default.js";
 
-describe('Provisioner', function () {
-	describe('Docker Image', function () {
-		it('Should create a GQL client on call to connect', async function () {
-			this.timeout(30000);
-			const provisioner = getProvisioner(DEFAULT_HOST);
+describe("Provisioner", function () {
+  describe("Docker Image", function () {
+    it.skip("Should create a GQL client on call to connect", async function () {
```

Comment:
We've gotta skip this for now since we can't test the full provisioning w/ dagger-in-dagger. This provisioning codepath will currently be tested manually by running the other tests on the host (same as the rest of the sdks for now).

I could have sworn I made an issue for this, but I can't immediately find it. I'll make one and link to this either way.

Reply:
Issue here, mention this test: https://github.com/dagger/dagger/issues/3852